### ISSUE TITLE
SMR/SNR order

### ISSUE THREAD

**Issue by Vcomea** [bcgov/CCISS_Review#41]
In By-BEC and CCISS spatial edatopes are labeled SNR/SMR e.g. C4, B2, D6
Feedback from ecologists is that end users will be expecting to see SMR/SNR, e.g. 4C, 2B, 6D

**Comment by cmahony** [comment]
agreed. i always used to use SMR/SNR. for some reason we switched for CCISS. let's do it this way from now on.

**Comment by Vcomea** [comment]
Sounds good! I will make those changes in my summary materials.

**Comment by cour10eygrace** [comment]
this still needs to be updated in CCISS/CCISSspatial

**Comment by Vcomea** [comment]
And needs to be updated in Colin's feasibility change maps and Vanessa's CCISS results summaries

**Comment by cmahony** [comment]
i discussed this with Kiri a few weeks ago. 4C doesn't work in r code; need to have a letter first, hence C4. we should use C4 in CCISS products and explain to our users why it is this way.

**Comment by Vcomea** [comment]
Thanks Colin. To add to our FAQs!

**Comment by Vcomea** [comment]
Added to FAQ list
